astropy/utils/exceptions.py

```diff
@@ -78,3 +73,22 @@ def __repr__(self):
 
 
 NoValue = _NoValue()
+
+
+def __getattr__(name: str):
+    if name in ("ErfaError", "ErfaWarning"):
+        import warnings
+
+        warnings.warn(
+            f"Importing {name} from astropy.utils.exceptions was deprecated "
+            "in version 6.1 and will stop working in a future version. "
+            f"Instead, please use\nfrom erfa import {name}\n\n",
+            category=AstropyDeprecationWarning,
+            stacklevel=1,
```

Comment:
It's usually recommended to atleast use `stacklevel=2` in these warnings (ref https://github.com/python/cpython/issues/88462)

Reply:
True, but in the context of a module level `__getattr__`, and from my quick experiments, it doesn't seem to make any difference.

Reply:
I think this is OK as is, so will merge now. Thanks, @neutrinoceros!